packages/design-picker/src/index.ts

```diff
@@ -9,6 +10,7 @@ export {
 	getDesignUrl,
 	getDesignPreviewUrl,
 	isBlankCanvasDesign,
+	mShotOptions,
```

Comment:
Maybe we don't need to export it? And I just found that the original `mShotOptions` is also a function 😅

Reply:
Yes! `mShotOptions` is still being used by the existing design picker [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/design-picker/src/components/index.tsx#L37), so will be leaving it as is for the moment 🙂